config/configmaps/install-log-regexes-configmap.yaml

```diff
@@ -6,17 +6,27 @@ metadata:
 data:
   regexes: |
     # AWS Specific
+    - name: AWSNATGatewayLimitExceeded
+      searchRegexStrings:
+      - "NatGatewayLimitExceeded"
+      installFailingReason: AWSNATGatewayLimitExceeded
+      installFailingMessage: AWS NAT gateway limit exceeded
+    - name: AWSVPCLimitExceeded
+      searchRegexStrings:
+      - "VpcLimitExceeded"
+      installFailingReason: AWSVPCLimitExceeded
+      installFailingMessage: AWS VPC limit exceeded
+    - name: LimitExceeded
```

Comment:
nit: Can we add an comment here that all limit exceeded regexes must be above this entry to allow specific matches before catch-all.